tests/test_cases/expected_kdl/multiline_string_double_backslash.kdl

```diff
@@ -0,0 +1 @@
+node "a\\ b\na\\b"
```

Comment:
this makes me _uncomfortable_

Reply:
oh. Oh no it's fine. I misread it. For some reason I thought the string ended with a bell (`\b`). But no, this is fine.